### PR TITLE
Added History Back to Talib Indicators 

### PR DESCRIPTION
**Pull request by RaedShabbir**
History allows for the calculation of ta indicators on all the previously seen data rather than just the window size of observations passed in. The observation columns method has been included as a way to ensure the observation_space for the trading environment is correctly set.

### CODE REVIEW

**Comment by Kukunin** [comment]
Does it make sense to implement the history in `feature_pipeline.py` instead of in a specific transformer? And pass it as a second argument to a transformer? This way it will be possible to implement other transformers based on previous observations.

Also, it'd be cool to add a `min_history_size` option so the environment doesn't pass observations to a model until N observations are collected and TA-lib, for example, will start to calculate adequate indicators.

**Comment by RaedShabbir** [comment]
These are great suggestions, will incorporate.

File: tensortrade/features/scalers/min_max_normalizer.py [hunk 1]
```diff
@@ -61,11 +73,10 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
             if high - low == 0:
                 normalized_column = (1/len(X[column])) * scale
             else:
-                normalized_column = (X[column] - low) / (high - low) * scale
-
+                normalized_column = np.abs((X[column] - low) / (high - low) * scale)
```
Comment (rhamnett) [inline tensortrade/features/scalers/min_max_normalizer.py]: need to import numpy

**Comment by notadamking** [comment]
These features have been removed from tensortrade.

**Pull request closed without merging**